app/models/jsonrates.rb

```diff
@@ -0,0 +1,46 @@
+require 'open-uri'
+require 'json'
+
+class Jsonrates
+  ENDPOINT_API = 'http://apilayer.net/api/'
+  ACCESS_KEY = "access_key=#{ENV['ENDPOINT_API_KEY']}"
+
+  attr_accessor :action, :currencies
+
+  def initialize(action = :live)
+    @action = action
+    @currencies = []
+  end
+
+  def load
+    return "Don't support this action yet!" unless check_api_support
+    stream = open(url_with_access_key)
+    JSON.parse(stream.read)
+  end
+
+  def build_url
+    url = "#{ENDPOINT_API}#{@action}?"
```

Comment:
[![Documentation](https://www.pullreview.com/assets/documentation-small.png)](https://www.pullreview.com/github/fabiosammy/dev_jsonrates/reviews/issue-3) Add documentation to the following method[ `Jsonrates#build_url`](https://www.pullreview.com/github/fabiosammy/dev_jsonrates/reviews/issue-3)